types/next/error.d.ts

```diff
@@ -10,5 +10,5 @@ export interface DefaultErrorIProps {
 }
 
 export default class Error<P = {}> extends React.Component<P & DefaultErrorIProps> {
-    static getInitialProps(context: NextContext): Promise<DefaultErrorIProps> | DefaultErrorIProps;
+    static getInitialProps: GetInitialProps<NextContext, DefaultErrorIProps>;
```

Comment:
Did you mean for this to go the other way around? (`GetInitialProps<IP, C>`)

Reply:
Oops, I should not have done this kind of mistake. Fixed.

Reply:
It's fine! And thanks for fixing.